README.md

```diff
@@ -34,20 +23,20 @@ Big thanks to @axyr for the initialisation of this module and his initial work.
  * @property int $Sort
  * @property int $Version
  * @property int $AuthorID
- * @method Member Author()
- * @method DataList|Category[] Categories()
- * @method ManyManyList|Tag[] Tags()
+ * @method \SilverStripe\Security\Member Author()
+ * @method \SilverStripe\ORM\DataList|Category[] Categories()
+ * @method \SilverStripe\ORM\ManyManyList|Tag[] Tags()
  * @mixin Versioned
  */
-class NewsItem extends DataObject
+class NewsItem extends \SilverStripe\ORM\DataObject
 {
     private static $db = array(
         'Title'	=> 'Varchar(255)',
         'Sort'	=> 'Int'
     );
 
     private static $has_one = array(
-        'Author'    => 'Member'
+        'Author'    => 'SilverStripe\Security\Member'
```

Comment:
SilverStripe generally encourages `use F\Q\C\Name` imports over inlining them - the same goes for PHPDoc declarations e.g. prefer `Member::class` in this line, and `@method Member Author()` in 26 above

Reply:
I've been going through most of them, missed this one apparently